android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothYunmaiSE_Mini.java

```diff
@@ -60,12 +60,13 @@ protected boolean onNextStep(int stepNr) {
                 final ScaleUser selectedUser = OpenScale.getInstance().getSelectedScaleUser();
                 byte sex = selectedUser.getGender().isMale() ? (byte)0x01 : (byte)0x02;
                 byte display_unit = selectedUser.getScaleUnit() == Converters.WeightUnit.KG ? (byte) 0x01 : (byte) 0x02;
+                byte body_type = (byte) YunmaiLib.toYunmaiActivityLevel(selectedUser.getActivityLevel());
 
                 byte[] user_add_or_query = new byte[]{
                         (byte) 0x0d, (byte) 0x12, (byte) 0x10, (byte) 0x01, (byte) 0x00, (byte) 0x00,
                         userId[0], userId[1], (byte) selectedUser.getBodyHeight(), sex,
                         (byte) selectedUser.getAge(), (byte) 0x55, (byte) 0x5a, (byte) 0x00,
-                        (byte)0x00, display_unit, (byte) 0x03, (byte) 0x00};
+                        (byte)0x00, display_unit, body_type, (byte) 0x00};
```

Comment:
If body_type is only 0 or 1 then why I wrote 3 here? Do you really know that this byte is for the body type? Unfortunately I can't test it anymore.

Reply:
I check this part in original app. and yes, this byte is body type.
imo you've been copied this info from [gist](https://gist.github.com/conoro/f0c1d96c450a8f5cce70e2846c3686c4)
and.. I don't know how conoro and pwnall get the 0x03 value

Reply:
ok I see thanks